lahja/tools/benchmark/process.py

```diff
@@ -45,30 +49,56 @@ def stop(self) -> None:
         self._process.terminate()
         self._process.join(1)
 
-    @staticmethod
-    def launch(config: DriverProcessConfig) -> None:
+    @classmethod
+    def launch(cls, config: DriverProcessConfig) -> None:
         # UNCOMMENT FOR DEBUGGING
         # logger = multiprocessing.log_to_stderr()
         # logger.setLevel(logging.INFO)
-        config.backend.run(DriverProcess.worker, config)
+        config.backend.run(cls.worker, config)
```

Comment:
Not part of this PR but: now that this method has been slimmed down so much `start()` could directly call `worker()`